### ISSUE TITLE
FAQ: Vaccination certificate QR code unreadable with white text on black background

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#1422]
## Where to find the issue

https://www.coronawarn.app/en/faq/ and
https://www.coronawarn.app/de/faq/

## Describe the issue

There is no warning in the FAQs that an EU Digital Covid Vaccination Certificate (PDF format) viewed online with viewing changed to white text on black background will not scan. I don't believe this is a CWA bug because a standard QR code scanner will not scan in the code either.

See the problem reported in https://github.com/corona-warn-app/cwa-app-android/issues/3514#issuecomment-867612726. Also commented on by @vaubaehn .

### Steps to reproduce

1. Open EU Digital Covid Vaccination Certificate (PDF format) in Adobe Acrobat Reader DC
2. On Start Screen in "Add Vaccination Certificate" click ADD
3. The certificate can be added.
4. In Adobe Acrobat Reader DC > Edit > Preferences > Accessibility
5. Select   Custom Color: Page Background: (black) Document Text (white)
6. Repeat step 2. to attempt to add the certificate
7. Nothing happens. The screen stays on the QR Code Scan layout.

## Suggested change

Add a new FAQ article describing the problem that if an EU Digital Covid Vaccination Certificate (PDF format) is viewed online with an application which displays the background as black or exchanges black and white colors, that the QR code cannot be scanned. The solution is to set the text to black and the background to white in the viewing application.

**Comment by MikeMcC399** [comment]
This is a weakness of the certificate design. The QR code should be surrounded by a white frame which is independent of the background of the rest of the document.

This should probably be fed back to the designers. What's the best way to do this?

**Comment by MikeMcC399** [comment]
The example Belgian certificate PDF **does** include a white frame around the QR code.
https://www.corona-tracking.info/wp-content/uploads/2021/05/EU-Digital-COVID-Certificates.pdf

**Comment by MikeMcC399** [comment]
Templates have been published to https://github.com/Digitaler-Impfnachweis/certification-apis/tree/master/templates

They do not however show the creation of the QR code, only its placement, so we should monitor if newer vaccination PDFs are being created with a white margin or not.

**Comment by MikeMcC399** [comment]
Thanks to the tip from @Ein-Tim in https://github.com/corona-warn-app/cwa-documentation/issues/671#issuecomment-888577328 I got a new version of my 2/2 vaccination certificate from the Bavarian portal https://impfzentren.bayern/citizen/ which uses the updated layout (QR is to the right of the text "Scannen Sie den nebenstehenden ..." instead of to the left, the QR is also larger: 50mm instead of 45mm).

The QR code is now surrounded with a white margin, so even if a PDF file is viewed with black background, the QR code is still readable by a QR code reader.

Closing this issue as I don't think that an FAQ article is now needed, since the format of the certificate has been improved and made more robust.
